recipes/ccache/all/conanfile.py

```diff
@@ -1,10 +1,10 @@
-from conan import ConanFile
+from conan import ConanFile, conan_version
```

Comment:
```suggestion
from conan import ConanFile
```
No longer needed, please remove it.